### ISSUE TITLE
UI flickering when adding more than one demod module

### ISSUE THREAD

**Issue by invader-zimm** [cropinghigh/sdrpp-inmarsatc-demodulator#1]
When I have more than one inmarsat demod module added in SDR++ and at least one of them is expanded, the left side of the UI is flickering significantly. Screen recording attached.

https://user-images.githubusercontent.com/76672128/188496157-5bb1cfb5-ed9a-4707-b002-0ff9e83dd0c3.mp4

**Comment by cropinghigh** [comment]
Currently can't say, why could this happen. There aren't any errors in the terminal?

**Comment by invader-zimm** [comment]
No, there is nothing printed out. No errors or warnings or any messages.
Maybe it has something to do with the constellation widget?

**Comment by cropinghigh** [comment]
Possible. You could try this by adding two meteor demodulators in same way.

**Comment by invader-zimm** [comment]
It might be related to that widget. Something strange is happening. I have:
- 2 inmarsat modules
- 1 meteor module

With this configuration, the inmarsat modules flicker.
If I add one more meteor or inmarsat demod, the flickering is gone...

**Comment by cropinghigh** [comment]
Wait, could you please try to decrease window size a bit to make it non-fullscreen?

**Comment by invader-zimm** [comment]
I'm using i3 so the window is always maximized. But if I make the window floating, then change the height, the flickering disappears.

**Comment by cropinghigh** [comment]
My thoughts were correct. For some reason when scrolling bar appears, size of menu decreases, and it immediately disappears, and so on forever. If you delete all of inmarsatc modules and still able to get that bug with one-two meteor decoder modules, report this as bug in the SDR++'s widget.

**Comment by invader-zimm** [comment]
I cannot reproduce with the meteor demod, only with the inmarsat demod. And even when the inmarsat demod flickers, the meteor one does not.

**Comment by cropinghigh** [comment]
I think i've found an issue. 
https://github.com/cropinghigh/sdrpp-inmarsatc-demodulator/blob/master/src/gui_widgets.h#L19
This thing calculates element width inside the function instead of taking it as argument, as in all other widgets.

**Comment by cropinghigh** [comment]
Try again with that commit  2d15436

**Comment by invader-zimm** [comment]
Unfortunately the flickering still occurs...
